docker/service/src/main/java/uk/nhs/adaptors/scr/mappings/from/fhir/RiskToPatientMapper.java

```diff
@@ -0,0 +1,43 @@
+package uk.nhs.adaptors.scr.mappings.from.fhir;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.hl7.fhir.r4.model.DateTimeType;
+import org.hl7.fhir.r4.model.Observation;
+import org.hl7.fhir.r4.model.Period;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.nhs.adaptors.scr.exceptions.FhirValidationException;
+import uk.nhs.adaptors.scr.models.xml.RiskToPatient;
+import static uk.nhs.adaptors.scr.utils.DateUtil.formatDateToHl7;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class RiskToPatientMapper {
+
+    public RiskToPatient map(Observation observation) {
+        var obj = new RiskToPatient();
+        obj.setIdRoot(observation.getIdentifierFirstRep().getValue());
+        var codingFirstRep = observation.getCode().getCodingFirstRep();
+
+        obj.setCodeCode(codingFirstRep.getCode());
+        obj.setCodeDisplayName(codingFirstRep.getDisplay());
+        obj.setStatusCodeCode("final");
+
+        if (observation.getEffective() instanceof DateTimeType) {
```

Comment:
i think we should pull this into a base class / util class at some point. If we can figure out the different object need?